### ISSUE TITLE
Error after indexing.

### ISSUE THREAD

**Issue by ItJustFox** [bpatrik/pigallery2#457]
**Describe the bug**

If you add any image with in emoji in the name (every emoji will do the bug) the index will simplely bug and stop it.

**Photo/video (optional) that causes the bug**

Any image with a emoji in the name

**Screenshots (optional)**

**Server logs (optional)**

```
QueryFailedError: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: '\xF0\x9F\x90\xBE\xE3\x81...' for column `PiGallery2`.`media_entity`.`name` at row 1
at new QueryFailedError (/app/node_modules/typeorm/error/QueryFailedError.js:12:28)

at Query.<anonymous> (/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:217:45)

at Query.<anonymous> (/app/node_modules/mysql/lib/Connection.js:526:10)
at Query._callback (/app/node_modules/mysql/lib/Connection.js:488:16)
at Query.Sequence.end (/app/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
at Query.ErrorPacket (/app/node_modules/mysql/lib/protocol/sequences/Query.js:92:8)
at Protocol._parsePacket (/app/node_modules/mysql/lib/protocol/Protocol.js:291:23)
at Parser._parsePacket (/app/node_modules/mysql/lib/protocol/Parser.js:433:10)
at Parser.write (/app/node_modules/mysql/lib/protocol/Parser.js:43:10)
at Protocol.write (/app/node_modules/mysql/lib/protocol/Protocol.js:38:16)
at Socket.<anonymous> (/app/node_modules/mysql/lib/Connection.js:88:28)
at Socket.<anonymous> (/app/node_modules/mysql/lib/Connection.js:526:10)
at Socket.emit (events.js:376:20)
at addChunk (internal/streams/readable.js:309:12)
at readableAddChunk (internal/streams/readable.js:284:9)
at Socket.Readable.push (internal/streams/readable.js:223:10)
at TCP.onStreamRead (internal/stream_base_commons.js:188:23) {
code: 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD',
errno: 1366,
sqlMessage: "Incorrect string value: '\\xF0\\x9F\\x90\\xBE\\xE3\\x81...' for column `PiGallery2`.`media_entity`.`name` at row 1",
sqlState: '22007',
index: 0,
sql: "INSERT INTO `media_entity`(`id`, `name`, `type`, `directoryId`, `metadataCaption`, `metadataCreationdate`, `metadataFilesize`, `metadataKeywords`, `metadataRating`, `metadataOrientation`, `metadataPersons`, `metadataBitrate`, `metadataDuration`, `metadataSizeWidth`, `metadataSizeHeight`, `metadataCameraDataIso`, `metadataCameraDataModel`, `metadataCameraDataMake`, `metadataCameraDataFstop`, `metadataCameraDataExposure`, `metadataCameraDataFocallength`, `metadataCameraDataLens`, `metadataPositionDataCountry`, `metadataPositionDataState`, `metadataPositionDataCity`, `metadataPositionDataGPSDataLatitude`, `metadataPositionDataGPSDataLongitude`, `metadataPositionDataGPSDataAltitude`) VALUES (DEFAULT, 'Pixiv PlzItsPrivate by tuplute🐾お仕事募集中.jpg', 'PhotoEntity', 21, DEFAULT, 1640427585000, 85771, DEFAULT, DEFAULT, 1, DEFAULT, DEFAULT, DEFAULT, 588, 884, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT)",
query: 'INSERT INTO `media_entity`(`id`, `name`, `type`, `directoryId`, `metadataCaption`, `metadataCreationdate`, `metadataFilesize`, `metadataKeywords`, `metadataRating`, `metadataOrientation`, `metadataPersons`, `metadataBitrate`, `metadataDuration`, `metadataSizeWidth`, `metadataSizeHeight`, `metadataCameraDataIso`, `metadataCameraDataModel`, `metadataCameraDataMake`, `metadataCameraDataFstop`, `metadataCameraDataExposure`, `metadataCameraDataFocallength`, `metadataCameraDataLens`, `metadataPositionDataCountry`, `metadataPositionDataState`, `metadataPositionDataCity`, `metadataPositionDataGPSDataLatitude`, `metadataPositionDataGPSDataLongitude`, `metadataPositionDataGPSDataAltitude`) VALUES (DEFAULT, ?, ?, ?, DEFAULT, ?, ?, DEFAULT, DEFAULT, ?, DEFAULT, DEFAULT, DEFAULT, ?, ?, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT)',
parameters: [
'Pixiv PlzItsPrivate by tuplute🐾お仕事募集中.jpg',
'PhotoEntity',
21,
```

**Environment (please complete the following information):**
 - OS: UnraidOS - Using the template for DockerCompose
 - Browser Chrome 

**Used app version**:
bpatrik/pigallery2:latest-debian-buster

**Comment by bpatrik** [comment]
Hi this should be fixed with #399
It should not be an issue with the nightly.

There are still issues with supporting emojis keywords and other parts, but should work with filenames.

**Comment by Railline** [comment]
> Hi this should be fixed with #399 It should not be an issue with the nightly.
> 
> There are still issues with supporting emojis keywords and other parts, but should work with filenames.

yeah it's fix in nightly OMG you app is so clean with all this function. thanks you for your app !
